### DISCUSSION TITLE
React app absolute import paths - NODE_PATH

### DISCUSSION THREAD

**Discussion by dkoleary88** [vercel/vercel#4240] [General]
I have a React app and for development convenience use absolute import paths that are specified in jsconfig.json:
```json
{
  "compilerOptions": {
    "baseUrl": "src"
  },
  "include": ["src"]
}
```

Build keeps failing in Vercel as modules cannot be found. It doesnt seem to respect jsconfig.json.

I've also tried setting the `NODE_PATH` env variable in various ways with no luck. 

Any ideas on how to solve this?

**Comment by coetry** [comment]
Hi @dkoleary88 can you give us some more context? Maybe a small repo that you can link to us? Are you using [babel-plugin-module-resolver](https://www.npmjs.com/package/babel-plugin-module-resolver)?
